moveit_core/robot_state/src/conversions.cpp

```diff
@@ -130,19 +130,19 @@ static inline void _robotStateToMultiDOFJointState(const RobotState& state, sens
   const std::vector<const JointModel*>& js = state.getRobotModel()->getMultiDOFJointModels();
   mjs.joint_names.clear();
   mjs.transforms.clear();
-  for (std::size_t i = 0; i < js.size(); ++i)
+  for (auto model : js)
```

Comment:
It took me a moment to understand that `auto&` would be redundant here because it's a primitive `JointModel*` ...